inventory.yml

```diff
@@ -0,0 +1,49 @@
+images:
```

Comment:
We could have TravisCI generate this in a pretest step and exclude it from the commit.

Alternatively, including it in the commit gives us an easy way to verify that the tests were run against a specific image without digging through the TravisCI output.

I see value in both.